tests/zkevm/test_worst_compute.py

```diff
@@ -105,6 +114,94 @@ def test_worst_keccak(
     )
 
 
+@pytest.mark.zkevm
```

Comment:
Hey, the `zkevm` marker is now automatically to all tests under `./tests/zkevm`, see:
- #1534
```suggestion
```
It can also be removed below.

Reply:
Cool, I'll remove the mark in all existing places. Thank you!